CONTRIBUTING.rst

```diff
@@ -0,0 +1,111 @@
+.. highlight:: shell
+
+============
+Contributing
+============
+
+Contributions are welcome, and they are greatly appreciated! Every
+little bit helps, and credit will always be given.
+
+You can contribute in many ways:
+
+Types of Contributions
+----------------------
+
+Report Bugs
+~~~~~~~~~~~
+
+Report bugs at https://github.com/Pylons/venusian/issues.
+
+If you are reporting a bug, please include:
+
+* Your operating system name and version.
+* Any details about your local setup that might be helpful in troubleshooting.
+* Detailed steps to reproduce the bug.
+
+Fix Bugs
+~~~~~~~~
+
+Look through the GitHub issues for bugs. Anything tagged with "bug"
+is open to whoever wants to implement it.
+
+Implement Features
+~~~~~~~~~~~~~~~~~~
+
+Look through the GitHub issues for features. Anything tagged with "feature"
+is open to whoever wants to implement it.
+
+Write Documentation
+~~~~~~~~~~~~~~~~~~~
+
+venusian could always use more documentation, whether as part of the
+official venusian docs, in docstrings, or even on the web in blog posts,
+articles, and such.
+
+Submit Feedback
+~~~~~~~~~~~~~~~
+
+The best way to send feedback is to file an issue at
+https://github.com/Pylons/venusian/issues.
+
+If you are proposing a feature:
+
+* Explain in detail how it would work.
+* Keep the scope as narrow as possible, to make it easier to implement.
+* Remember that this is a volunteer-driven project, and that contributions
+  are welcome :)
+
+Get Started!
+------------
+
+Ready to contribute? Here's how to set up `venusian` for local development.
+
+1. Fork the `venusian` repo on GitHub.
+2. Clone your fork locally::
+
+    $ git clone git@github.com:your_name_here/venusian.git
+
+3. Install your local copy into a virtualenv::
+
+    $ python3 -m venv env
+    $ env/bin/pip install -e .[docs,testing]
+    $ env/bin/pip install tox
+
+4. Create a branch for local development::
+
+    $ git checkout -b name-of-your-bugfix-or-feature
```

Comment:
```suggestion
    $ git switch -c name-of-your-bugfix-or-feature
```

Using `git checkout` is not recommend by `git` anymore due to confusion related to what it means to do a `checkout`.

Reply:
Can you point me to a reference for that recommendation?  The [basic branching and merging chapter](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging) of the official git book still uses `checkout -b`.

Reply:
https://linuxhandbook.com/git-switch-checkout/

https://git-scm.com/docs/git-switch

git switch was explicitly added to stop confusion around what "checkout" does since checkout was overloaded.